File: PSP/GachaponFX/src/main/java/modelo/Personaje.java
package modelo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Personaje {
    private String nombre;
    private int rareza;
    private int precio;
    private int id;
    private String icono;
}

File: Moviles/RecyclerView/app/src/main/java/com/example/recyclerview/domain/modelo/Personaje.kt
package com.example.recyclerview.domain.modelo

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.parcelize.Parcelize

@JsonClass(generateAdapter = true)
@Parcelize
data class Personaje(
    @Json(name = "nombre")
    val nombre : String,
    @Json(name = "rareza")
    val rareza : Int,
    @Json(name = "precio")
    val precio : Int) : Parcelable